### PR TITLE
Replace master with main in links and docs/conf.py

### PR DESCRIPTION
**Pull request by svekars**
Fixes #ISSUE_NUMBER

cc @carljparker

### CODE REVIEW

File: docs/source/fx.rst [hunk 1]
```diff
@@ -1039,7 +1039,7 @@ Miscellanea
         traced.eval()
 
         x = torch.randn(5, 3)
-        torch.testing.assert_close(traced(x), x)
+        torch.testing.assert_allclose(traced(x), x)
```
Comment (albanD) [inline docs/source/fx.rst:1042]: WHy?

File: docs/source/jit.rst [hunk 2]
```diff
@@ -161,7 +161,7 @@ Example (using a traced module):
 
     class MyScriptModule(torch.nn.Module):
         def __init__(self):
-            super().__init__()
+            super(MyScriptModule, self).__init__()
```
Comment (albanD) [inline docs/source/jit.rst:164]: Is this due to a bad merge? We definitely don't want that.

File: docs/source/jit.rst [hunk 3]
```diff
@@ -831,14 +831,19 @@ New API:
 
 ::
 
-    from typing import Final
+    try:
```
Comment (albanD) [inline docs/source/jit.rst:834]: Why not the version from typing?

File: docs/source/notes/modules.rst [hunk 4]
```diff
@@ -599,7 +599,7 @@ PyTorch provides two types of hooks for modules:
 * **Forward hooks** are called during the forward pass. They can be installed for a given module with
   :func:`~torch.nn.Module.register_forward_pre_hook` and :func:`~torch.nn.Module.register_forward_hook`.
   These hooks will be called respectively just before the forward function is called and just after it is called.
-  Alternatively, these hooks can be installed globally for all modules with the analogous
+  Alternatively, these hooks can be installed globally for all modules with the analagous
```
Comment (albanD) [inline docs/source/notes/modules.rst:602]: no?

**Review by albanD — CHANGES_REQUESTED** [review]
I want to ask "why this is not pointing to stable instead of main" for a bunch of these. But that can be a separate conversation.

Also I feel like this PR is reverting some recent cleanups that happened in the doc. We should not do that.

File: docs/source/fx.rst (hunk 1 above)
Comment (svekars) [reply docs/source/fx.rst]: fixed.

**Review by albanD — APPROVED** [review]
Change sounds ok.
What's the plan to test that it doesn't break nightly / release?

**Review by malfet — APPROVED** [review]

**Pull request closed without merging**